src/utils/flags.ts

```diff
@@ -377,6 +377,11 @@ export const withDeploymentFlowFlag = flags.integer({
   description: 'deployment flow ID',
 })
 
+export const withEntityTypeFlag = flags.string({
```

Comment:
Do you imagine this flag being used outside of the `entities` topic? We should talk about where purpose-specific flags ought to go, or this file risks becoming bloated with time.

Reply:
It's going to be some time before the V4 API gets expanded so I think the flags file will remain at the size it currently is. If and when we do get an expansion of the V4 API and start adding new commands, then yeah, it will be something to consider.